modules/components/analytics/web/org.wso2.carbon.pc.analytics.web/src/web/js/actions.js

```diff
@@ -788,16 +883,20 @@ function drawTotalInvolvedInstanceCountVsProcessIdResult(renderElement){
                 alert(errorJson.message);
             }
         });
-    }else{
+    } else {
         console.log('Empty user id list.');
```

Comment:
Using console.log in JS will only print the message in browser console. We need to append the message into our logs and need to display the message in UI.